RELEASE-NOTES.txt

```diff
@@ -3,8 +3,12 @@
 19.9
 -----
 * [*] Editor: Improved editor performance by reducing memory usage [https://github.com/wordpress-mobile/WordPress-Android/pull/16490]
-
 * [*] Block Editor: Prevent non-functional '+' icon displaying in toolbar on self-hosted sites [https://github.com/wordpress-mobile/WordPress-Android/pull/16507]
+* [**] Block Editor: Cover Block: Improve color contrast between background and text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4808]
+* [**] Block Editor: Buttons block: Fix issue related to displaying formatting buttons after closing the block settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4814]
+* [***] Block Editor: Add drag & drop blocks feature [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4832]
+* [*]  Block Editor: Gallery block: Fix broken "Link To" settings and add "Image Size" settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4841]
+* [*]  Block Editor: Unsupported Block Editor: Prevent WordPress.com tour banner from displaying. [https://github.com/wordpress-mobilegutenberg-mobile/pull/4820]
```

Comment:
I found this typo: `wordpress-mobilegutenberg-mobile` => `wordpress-mobile/gutenberg-mobile`

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/16522/commits/a548383fd1082bc213491a14aadaf85c11708e86.